File: Iniciante/PHP/exercise_1074.php
<?php

$quantidade = readline();
$valores = [];

for ($i = 0; $i < $quantidade; $i++) {
    array_push($valores, readline());
}

foreach ($valores as $valor) {
    $resultado = '';

    if ($valor == 0) {
        print_r("NULL\n");
    } else {
        if ($valor%2 == 0) {
            $resultado.="EVEN";
        } else {
            $resultado.="ODD";
        }

        if ($valor > 0) {
            $resultado.=" POSITIVE";
        } else {
            $resultado.=" NEGATIVE";
        }

        print_r($resultado."\n");
    }
}

File: Iniciante/Python/exercise_1074.py
number = int(input())
numbers = []

for x in range(number):
    numbers.append(int(input()))

for newNumber in numbers:
    if newNumber == 0:
        print('NULL')
    else:
        if newNumber % 2 == 0:
            if newNumber > 0:
                print('EVEN POSITIVE')
            elif newNumber < 0:
                print('EVEN NEGATIVE')
        else:
            if newNumber > 0:
                print('ODD POSITIVE')
            elif newNumber < 0:
                print('ODD NEGATIVE')